requirements/requirements.txt

```diff
@@ -12,4 +12,5 @@ wrapt
 ruamel.yaml
 scikit-learn
 scipy
+frozendict
```

Comment:
looks like this has to go to requirements_test.txt. Also, if you keep it here then please remove from requirements_asr.txt

Reply:
oh yeah, thanks